src/parser/postgresparser.cpp

```diff
@@ -1016,6 +1016,11 @@ parser::SQLStatement *PostgresParser::CreateTransform(CreateStmt *root) {
         }
         // Update Reference Table
         col->fk_sink_table_name = constraint->pktable->relname;
+        if (constraint->pktable->schemaname) {
```

Comment:
Maybe more comments over here specifically about temporary tables in addition to the general description. Not clear about why adding temporary tables needs this.